config/doxx.coffee

```diff
@@ -13,7 +13,8 @@ module.exports = {
   templatesDir: config.templatesDir
   partialsDir: config.partialsDir
   metaExtra: (fileName) ->
-    improveDocsLink: "#{config.editPageLink}/#{config.docsSourceDir}/#{fileName}"
+    [..., newName] = fileName.split "/"
```

Comment:
Nit: the convention normally is brackets for single-line calls, omitted for multi-line calls, and single quotes for strings, with `"` only for strings using interpolation (`#{}`). Docs here: https://github.com/resin-io/process/blob/master/process/coffeescript-coding-standards.md. That means this would be `fileName.split('/')`